packages/content-source-maps/vite.config.ts

```diff
@@ -9,6 +9,7 @@ export default defineConfig({
     exclude: [...configDefaults.exclude],
   },
   build: {
+    target: 'es2017',
```

Comment:
we have here now `es2017` as target and on `.swrc` we have `es2020`, can this lead to issues?

Reply:
i think we are not using swc at all, i had made a change to the .swrc file but the build target didn't change. this must have been left over at some point

Reply:
maybe for a follow up, wonder if we can keep es2020 for `es` format and only use `es2017` for cjs? 🤔